src/main/java/com/commercetools/project/sync/service/impl/CustomObjectServiceImpl.java

```diff
@@ -28,6 +30,8 @@
   private static final long MINUTES_BEFORE_CURRENT_TIMESTAMP = 2;
 
   private SphereClient sphereClient;
+  private String runnerName = DEFAULT_RUNNER_NAME;
+  private String methodName = StringUtils.EMPTY;
```

Comment:
I don't mind personally either `""` or  `StringUtils.EMPTY`. But what I think we should do is be consistent. So use only of them so it's not confusing why sometimes we use this and why the other. 

https://github.com/commercetools/commercetools-project-sync/pull/30/files#diff-90d173bf2f68b44092d0e0fa23e07f36R29 we use ""

Reply:
6061a6b87f32a769d39e69230a7f08217afd373b